docs/painless/painless-getting-started.asciidoc

```diff
@@ -136,8 +136,9 @@ the document, `doc['field'].value` for this document returns:
 IMPORTANT: Starting in 7.0, `doc['field'].value` throws an exception if
 the field is missing in a document. To enable this behavior now,
 set a {ref}/jvm-options.html[`jvm.option`] 
-`-Des.scripting.exception_for_missing_value=true` on a node. If you do not enable
-this behavior, a deprecation warning is logged on start up.
+`-Des.scripting.exception_for_missing_value=true` on a node. If you do not
+enable this behavior, a one-time deprecation warning is logged for each
```

Comment:
I think this explanation needs to be updated given the use of deprecatedAndMaybeLog?